### ISSUE TITLE
Flagged as malware on Microsoft Edge

### ISSUE THREAD

**Issue by amirmah** [FelisCatus/SwitchyOmega#2410]
Hello,
Microsoft Edge has flagged this extension as malware. 
Do you know why?

**Comment by amirmah** [comment]
Oh, I think there was a fake extension with exactly same name and same logo on Microsoft Edge store, which Microsoft removed and flagged as malware today. the real extension is still available.
This was the URL of the fake one:
https://microsoftedge.microsoft.com/addons/detail/dijmmgblneagkcdganednabkbgjmceoe

**Comment by jachaussloef** [comment]
Yeah, what Microsoft Edge flagged is not official one. Someone used codes here and injected malware into it to compile the fake version you mentioned above.
You should install from https://chrome.google.com/webstore/detail/proxy-switchyomega/padekgcemlokbadohgkifijomclgjgif

**Comment by KexyBiscuit** [comment]
If you really need one from Microsoft Edge Addons, here's one I personally verified as safe: https://github.com/FelisCatus/SwitchyOmega/issues/2004#issuecomment-739812969

The specific version I have verified:
Version 2.5.20
Updated December 4, 2020
